package.json

```diff
@@ -20,15 +20,20 @@
         "@types/tmp": "^0.0.33",
         "clang-format": "1.0.49",
         "concurrently": "^3.5.1",
+        "http-server": "^0.11.1",
         "protobufjs": "5.0.0",
         "protractor": "^5.2.0",
         "tsickle": "0.25.x",
         "tsutils": "2.20.0",
         "typescript": "2.7.x"
     },
     "scripts": {
-        "pretest": "webdriver-manager update && bazel build examples/app:all",
-        "test": "concurrently \"bazel run examples/app:devserver\" protractor --kill-others --success first",
+        "pretest": "webdriver-manager update $CHROMEDRIVER_VERSION_ARG && bazel build --config=ci examples/app:e2e && bazel build --config=ci examples/protocol_buffers:e2e",
+        "test": "yarn test-examples-app-devserver && yarn test-examples-app-prodserver && yarn test-examples-protobuf-devserver && yarn test-examples-protobuf-prodserver",
```

Comment:
this is kinda gross - can't we keep the protractor config in the root and tell it about multiple tests to run?

Reply:
Good idea. I made it work with a single protractor config making use of `suites`. The serving port has to be the same so I added a `port` attribute to ts_devserver. Can't think of how to remove the multiple package.json scripts unless test is just changed to a shell script that run the tests serially.

Reply:
sure

Reply:
On second thought. If changed to a shell script then the e2e tests won't run on windows :(